src/CacheCow.Server/CachingHandler.cs

```diff
@@ -1,646 +1,654 @@
-﻿using System;
```

Comment:
Line endings mismatch. You can add ?w=1 to the URL on GitHub to ignore that... https://github.com/aliostad/CacheCow/pull/169/files?w=1